Proposals.md

```diff
@@ -51,7 +51,7 @@ You can learn more about contributing new proposals (and other ways to contribut
 | ------------------------------------------------------------------------------ | -------------------------------------- | -------- |
 | [Blob Store][wasi-blob-store]                                                  | Jiaxiao Zhou, Dan Chiarlone, David Justice |          |
 | [Crypto][wasi-crypto]                                                          | Frank Denis and Daiki Ueno             |          |
-| [Digital I/O][wasi-digital-io]                      | Emiel Van Severen |          |
+| [GPIO][wasi-gpio]                      | Merlijn Sebrechts, Michiel Van Kenhove |          |
```

Comment:
```suggestion
| [GPIO][wasi-gpio]                      | Merlijn Sebrechts, Maximilian Seidler |          |
```

Should be Maximilian instead of Michiel